src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <!-- img -->
+      <div class="card__img">
+        <picture>
+          <img
+            src="images/imac.jpeg"
+            alt="card-img"
+            loading="lazy"
+          >
+        </picture>
+      </div>
+
+      <!-- cont -->
+      <div class="card__cont">
+        <!-- head -->
+        <div class="card__head">
+          <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+          <div class="card__code">Product code: 195434</div>
+        </div>
+
+        <!-- stars -->
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p>Reviews: 5</p>
+        </div>
+
+        <!-- price -->
+        <div class="card__price">
+          <p>Price:</p>
+          <strong>$2,199</strong>
+        </div>
+
+        <!-- btn -->
+        <a
+          href="#"
+          class="card__btn"
+          data-qa="hover"
+        >Buy</a>
```

Comment:
follow code styling recommendations
```suggestion
        <a
          href="#"
          class="card__btn"
          data-qa="hover"
        >
          Buy
        </a>
```